keda/values.yaml

```diff
@@ -22,6 +22,7 @@ imagePullSecrets: []
 
 operator:
   name: keda-operator
+  useHostNetwork: false
```

Comment:
Should we define ` dnsPolicy` too as we do in the metrics server?
https://github.com/kedacore/charts/blob/main/keda/values.yaml#L40-L42

It seems that it's required to change it if we use `hostNetwork`

Reply:
By default when i install helm chart `dnsPolicy: ClusterFirst` got assigned